### PR TITLE
404 Fix, add info window animation, add frosted glass header

### PR DESCRIPTION
**Pull request by 1ceit**

### CODE REVIEW

File: css/main.sass [hunk 1]
```diff
@@ -351,10 +350,10 @@ main > img, .img-row, table
         height: 100%
 
 table img
-    width: 100%
+        width: 100%
 
 main h1
-    text-align: center
+        text-align: center
```
Comment (ErikBoesen) [inline css/main.sass:356]: Not sure why all the whitespace changes but not really a big deal

File: 404.html [hunk 2]
```diff
@@ -22,10 +22,10 @@
                 }
             }, 1000);
 
-            // Check if a phone or iPad -> @media tags cant change actual text only js can
-            if (window.innerWidth <= 700 ||(window.innerWidth > 700 && window.innerWidth <= 1024)) {
+            // Check if a phone or iPad -> @media tags cant change actual text only js
+            if (window.innerWidth <= 700 || (window.innerWidth > 700 && window.innerWidth <= 1024)) {
                 document.body.classList.add('mobile');
-
+
```
Comment (ErikBoesen) [inline 404.html:28]: nit: you should configure your IDE to remove trailing whitespace

File: js/alumni.js [hunk 3]
```diff
@@ -372,6 +374,8 @@ function initMap() {
                 '</a>' +
                 "</div>";
             infoWindow.setContent(contentString);
+            map.setZoom(5);
+            map.setZoom(7);
```
Comment (ErikBoesen) [inline js/alumni.js:378]: info: what is this doing exactly?

**Review by ErikBoesen — APPROVED** [review]

File: css/main.sass (hunk 1 above)
Comment (1ceit) [reply css/main.sass:356]: i had done alt + shift + f in habit

File: js/alumni.js (hunk 3 above)
Comment (1ceit) [reply js/alumni.js:378]: when focusing it zooms, there's a bit of a zoom-out then in, but it's not really noticeable

**Pull request merged**